### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cyrille-sg**
Making a much more friendly README for people that want to start to use the Go client

### CODE REVIEW

**Comment by Soulou** [comment]
There should be a link to the godoc, and the examples should be part of the code to be displayed as examples in the godoc too :) my 2cts

File: README.md [hunk 1]
```diff
@@ -2,7 +2,49 @@
...
+
+func PanicOnError(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+```
+## Explore
+
+As this go client maps all the public scalingo APIs, you can explore the [API documentation](https://developers.scalingo.com/)
```
Comment (EtienneM) [inline README.md:42]:
```suggestion
As this Go client maps all the public Scalingo APIs, you can explore the [API documentation](https://developers.scalingo.com/).
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,49 @@
...
+If you need an implementation example, you can take a look at the [Scalingo CLI code](https://github.com/Scalingo/cli)
```
Comment (EtienneM) [inline README.md:43]:
```suggestion
If you need an implementation example, you can take a look at the [Scalingo CLI code](https://github.com/Scalingo/cli).
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,49 @@
 
 # Go client for Scalingo API v4.5.4
 
-## Add Support for a New Event
+This repository is the Go client for the [Scalingo APIs](https://developers.scalingo.com/)
+
+## Getting started
```
Comment (EtienneM) [inline README.md:7]:
```suggestion
## Getting Started
```

File: README.md [hunk 4]
```diff
@@ -2,7 +2,49 @@
...
+		panic(err)
+	}
+}
+```
+## Explore
+
+As this go client maps all the public scalingo APIs, you can explore the [API documentation](https://developers.scalingo.com/)
+If you need an implementation example, you can take a look at the [Scalingo CLI code](https://github.com/Scalingo/cli)  
+
+## Repository processes
```
Comment (EtienneM) [inline README.md:45]:
```suggestion
## Repository Processes
```

File: README.md [hunk 5]
```diff
@@ -2,7 +2,49 @@
 
 # Go client for Scalingo API v4.5.4
 
-## Add Support for a New Event
+This repository is the Go client for the [Scalingo APIs](https://developers.scalingo.com/)
+
+## Getting started
+
+```
```
Comment (EtienneM) [inline README.md:9]:
```suggestion
```go
```

File: README.md [hunk 6]
```diff
@@ -2,7 +2,49 @@
...
+	client, err := getClient()
+	PanicOnError(err)
+	apps, err := client.AppsList()
+	PanicOnError(err)
+	for _, app := range apps {
+		println("App: " + app.Name)
+	}
+}
+
+func PanicOnError(err error) {
```
Comment (EtienneM) [inline README.md:34]: This is really not a Go idiomatic form. I wouldn't use it in a Go example.

File: README.md [hunk 7]
```diff
@@ -2,7 +2,49 @@
...
+```
+package main
+
+import (
+	"github.com/Scalingo/go-scalingo"
+)
+
+func getClient() (*scalingo.Client, error) {
+	config := scalingo.ClientConfig{
+		APIEndpoint: "https://api.osc-fr1.scalingo.com", // Possible endpoints can be found at https://developers.scalingo.com/
```
Comment (EtienneM) [inline README.md:18]:
```suggestion
		APIEndpoint: "https://api.osc-fr1.scalingo.com", // Possible endpoints can be found at https://developers.scalingo.com/#endpoints
```

**Comment by cyrille-sg** [comment]
@Soulou To appears properly in https://pkg.go.dev/github.com/Scalingo/go-scalingo?tab=overview, we will need to add a licence

File: README.md [hunk 8]
```diff
@@ -2,7 +2,47 @@
 
 # Go client for Scalingo API v4.5.4
 
-## Add Support for a New Event
+This repository is the Go client for the [Scalingo APIs](https://developers.scalingo.com/)
```
Comment (EtienneM) [inline README.md:5]:
```suggestion
This repository is the Go client for the [Scalingo APIs](https://developers.scalingo.com/).
```

**Comment by Soulou** [comment]
@cyrille-sg Can you add a BDS 3-clause license, like our CLI: https://github.com/Scalingo/cli/blob/master/LICENSE

Thanks!

**Pull request merged**